RobotCode/snobot2018/src/org/snobot/leds/SnobotLedManager.java

```diff
@@ -96,21 +120,27 @@ public void update()
         {
             updateLedsFromRobotState();
         }
+
         mLedStrip.updateStrip();
     }
 
     private void updateLedsFromRobotState()
     {
-        if (mRobot.isAutonomous())
+        if (mRobot.isDisabled())
```

Comment:
Allows us to update during disabled.  I don't think it is 100% clear when the game data comes through the DS, but this potentially can let us see if we need to change our auton start logic if we don't see the LEDS change